src/lib/reasoners/intervalCalculus.ml

```diff
@@ -1353,6 +1353,9 @@ let fm (module Oracle : S with type p = P.t) uf are_eq rclass_of env eqs =
 let is_num r =
   let ty = X.type_info r in ty == Ty.Tint || ty == Ty.Treal
 
+let is_num_term r =
+  let ty = E.type_info r in ty == Ty.Tint || ty == Ty.Treal
+
```

Comment:
We should use `is_num` function.

Reply:
Oups, I didn't notice it wasn't the same type.

Reply:
The fact that the term is called `r` doesn't help, to be fair :sweat_smile: